features/command_line/order.md

```diff
@@ -22,4 +25,5 @@ config file (e.g. `.rspec`) and you want to override it from the command line.
 --order rand
 --order rand:123
 --seed 123 # same as --order rand:123
+--modification_time
```

Comment:
Does this use the underscore by design?

Reply:
I think `modificationtime` would be ugly. Any suggestion?

Reply:
I was thinking that a hyphen to keep it in line with `--force-colour` et al.

`mtime` is also an option, but it's system-specific (*nix only). On the other hand, `File.mtime` is a Ruby API, and it works for Windows, too.

Did we discuss this in the ticket, and decided to go with "modification time" instead of "recently modified"? I may be mistaken as a non-native speaker, "modification time" wouldn't give me much clue if it's oldest first or most recently modified first.

Aside from this, https://github.com/rspec/rspec-core/issues/2762 suggests `--order modification_time`, and apart from the underscore, I'm in favour of using `--order`.
Please keep in mind that `--next-failure` seems to override `--order`, too. Maybe mentioning that `--only-failures` should be used instead, and/or making sure that flag order (`--order mtime --next-failure` vs `--next-failure --order mtime`) on the command line does not change the behaviour.
I'd add a couple of examples to `spec/integration/order_spec.rb` to solidify it.

Reply:
I picked `--order recently-modified` because to me it is clearer.